lib/_tls_wrap.js

```diff
@@ -1083,7 +1082,8 @@ function onConnectEnd() {
   if (!this._hadError) {
     const options = this[kConnectOptions];
     this._hadError = true;
-    const error = new Error('socket hang up');
+    const error = new Error('Client network socket disconnected before ' +
+                            'secure TLS connection was established');
```

Comment:
I can't remember if we're keeping this intentionally with the old errors or if this needs to be converted to use internal/errors

Reply:
I suggest to switch to the internal/errors and override the code for now. It is not a strong opinion though and I am fine with keeping it as it is for now as well.

Reply:
I think I’m going to leave this out of this PR.

I don’t think I’m a fan of changing the `code` property of exactly those errors where we always provided one, though.